include/complex_quda.h

```diff
@@ -25,7 +25,7 @@
 #include <cmath>
 #include <complex>
 #include <sstream>
-//#include <cuComplex.h>
+//#include <qudaComplex.h>
```

Comment:
Just delete this I guess.  Note eventually I probably want to do away with QUDA's custom complex class and just use `std::complex` when libcu++ ships support for this.  At that point, the onus would be on the target arch to provide its own version of `std::complex`.

Reply:
12a0b62cb